### DISCUSSION TITLE
We Been Waiting For The Next Update For A LONG Time.

### DISCUSSION THREAD

**Discussion by JaegerAxelRetro2000s** [ntsc-rs/ntsc-rs#245] [Ideas]
We Need Some Ideas Like A CRT Screen For The Update Or Maybe VHS Audio/Damage Effects?

**Comment by notdigidani** [comment]
i fr need to see what they have been working on, i want updates😔

**Comment by valadaptive** [comment]
Audio effects are being tracked in https://github.com/valadaptive/ntsc-rs/issues/15. Basically, there's already a good third-party plugin for it.

There were 3 months between 0.8.0 and 0.9.0. The next version will be ready when it's ready.

**Reply by JaegerAxelRetro2000s** [reply]
OMG Thank The NTSC-RS Lord!

**Reply by notdigidani** [reply]
![image](https://github.com/user-attachments/assets/8a2855cd-b501-41d4-b4f4-7caad7ff6aab)
